src/bindings/overlay.js

```diff
@@ -169,6 +183,10 @@ function isElementAllowed(element) {
  * @private
  */
 function isAttrAllowed(attr, element) {
+  // Does it have a namespace that matches the element's?
+  if (attr.ns === null || DOM_NAMESPACES[attr.ns] !== element.namespaceURI) {
```

Comment:
If you pass the DOM Attribute object as `attr` here, you can simply do:

    if (attr.namespaceURI !== element.namespaceURI)